ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/config/Config.java

```diff
@@ -47,4 +50,14 @@
   Optional<ClientTlsOptions> getClientTlsOptions();
 
   Collection<String> getCorsAllowedOrigins();
+
+  Boolean isMetricsEnabled();
+
+  Integer getMetricsPort();
+
+  String getMetricsHost();
+
+  Set<MetricCategory> getMetricCategories();
+
+  List<String> getMetricsHostAllowList();
```

Comment:
can probably just be "collection" - or do we need to iterate it in order?

Reply:
could be but the Besu MetricsConfiguration takes a list, so it's easier to just pass a list through all the way. instead of taking a collection and converting to list for the metrics service. internally it only processes the allow list as a collection.